src/utils/indexForSearch.ts

```diff
@@ -10,18 +10,36 @@ import { search } from "../lib/search"
  * This indexes an ElasticSearch cluster with collections data.
  */
 
+interface DebugRecord {
+  slug: string
+  reason: string
+  data: string
+}
+
 export const indexForSearch = async () => {
   let connection: Connection | undefined
-
+  const bogus_collections: DebugRecord[] = []
   try {
     const connectionArgs = databaseConfig()
     connection = await createConnection(connectionArgs)
 
     if (connection.isConnected) {
       const repository = getMongoRepository(Collection)
-
       const collections = await repository.find()
       for (const collection of collections) {
+        console.log(`Now processing ${collection.slug}`)
+
+        if (!collection.query) {
+          console.log(
```

Comment:
we have some stuff in the prod database with no queries defined - I think this happened when we deployed stub collection hubs with no membership leading to a null query, which is what has been causing the failure in our cron jobs. So this is the minimum fix to the current issue.